src/main/java/com/umc/networkingService/domain/todoList/controller/TodoListController.java

```diff
@@ -0,0 +1,80 @@
+package com.umc.networkingService.domain.todoList.controller;
+
+import com.umc.networkingService.config.security.auth.CurrentMember;
+import com.umc.networkingService.domain.member.entity.Member;
+import com.umc.networkingService.domain.todoList.dto.request.TodoListCreateRequest;
+import com.umc.networkingService.domain.todoList.dto.request.TodoListUpdateRequest;
+import com.umc.networkingService.domain.todoList.dto.response.TodoListGetResponse;
+import com.umc.networkingService.domain.todoList.dto.response.TodoListGetResponses;
+import com.umc.networkingService.domain.todoList.dto.response.TodoListIdResponse;
+import com.umc.networkingService.domain.todoList.service.TodoListService;
+import com.umc.networkingService.global.common.base.BaseResponse;
+import io.swagger.v3.oas.annotations.Operation;
+import io.swagger.v3.oas.annotations.responses.ApiResponse;
+import io.swagger.v3.oas.annotations.responses.ApiResponses;
+import io.swagger.v3.oas.annotations.tags.Tag;
+import lombok.RequiredArgsConstructor;
+import org.springframework.web.bind.annotation.*;
+
+import java.time.LocalDate;
+import java.util.UUID;
+
+@Tag(name = "TodoList API", description = "TodoList 관련 API")
+@RestController
+@RequiredArgsConstructor
+@RequestMapping("/to-do-lists")
+public class TodoListController {
+
+    private final TodoListService todoListService;
+
+    @Operation(summary = "투두리스트 작성 API", description = "투두리스트를 작성하는 API입니다.")
+    @ApiResponses(value = {
+            @ApiResponse(responseCode = "COMMON200", description = "성공")
+    })
+    @PostMapping
+    public BaseResponse<TodoListIdResponse> createTodoList(@CurrentMember Member member, @RequestBody TodoListCreateRequest request){
```

Comment:
TodoListCreateRequest 클래스를 확인해보니 title이라는 필드에 대해서 validation을 적용해줬기 때문에 @Valid 어노테이션을 붙여줘야 할 거 같아요
```suggestion
    public BaseResponse<TodoListIdResponse> createTodoList(@CurrentMember Member member, 
                                                                                                    @Valid @RequestBody TodoListCreateRequest request){
```

Reply:
수정했습니다.